src/envo/envo-edit.owl

```diff
@@ -41163,6 +41165,23 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105126> "weathering of plastic"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105126> <http://purl.obolibrary.org/obo/ENVO_06105119>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105201> (water table)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/w/index.php?title=Water_table&oldid=1049566132") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105201> "The water table is the surface where the water pressure head is equal to the atmospheric pressure."@en)
```

Comment:
Hi @KrishnaTO - please use the genus-differentia definitional form:
```
A B which Cs. 
```
Where B is the genus and C is the differentia (see suggested genus on line 41174)



```suggestion
AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/w/index.php?title=Water_table&oldid=1049566132") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105201> "An interface layer which 1) separates a phreatic zone from a vadose zone and 2) contains interstitial water with a pressure head equal to atmospheric pressure."@en)
```

You can add more axioms (e.g. adjacency) to link this to the zones noted above.